src/Mapper/AutoMapper.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+
+namespace App\Mapper;
+
+
+use Exception;
+
+class AutoMapper
```

Comment:
can you create interface for it, so you can inject it ,and may change the implementation in the future ?